src/ucp/core/ucp_mm.inl

```diff
@@ -78,6 +78,14 @@ ucp_memh_put(ucp_context_h context, ucp_mem_h memh, int invalidate)
                 (ucs_rcache_invalidate_comp_func_t)ucs_empty_function, NULL);
     }
 
+    if ((memh->super.flags & UCS_RCACHE_REGION_FLAG_PGTABLE) &&
+        (memh->super.refcount == 2) &&
```

Comment:
i'd define some constant instead of `2`, or plz add some comment explaining the value